BitwardenShared/Core/Platform/Services/AuthenticatorSyncService.swift

```diff
@@ -39,8 +42,8 @@ actor DefaultAuthenticatorSyncService: NSObject, AuthenticatorSyncService {
     /// The service used by the application to report non-fatal errors.
     private let errorReporter: ErrorReporter
 
-    /// Notification Center Service to subscribe to foreground/background events.
-    private let notificationCenterService: NotificationCenterService
```

Comment:
🤔 Is the notification center service getting removed because we don't actually need to know if the app is foregrounded/backgrounded?

Reply:
Yes. It was left over from a previous refactor (previously we were waiting for the foreground to take action).